build_keyboard.mk

```diff
@@ -196,6 +196,8 @@ ifeq ($(strip $(RGBLIGHT_ENABLE)), yes)
 	OPT_DEFS += -DRGBLIGHT_ENABLE
 	SRC += $(QUANTUM_DIR)/light_ws2812.c
 	SRC += $(QUANTUM_DIR)/rgblight.c
+    CIE1931_CURVE = yes
```

Comment:
There were mixed spaces and inside the whole file, so I converted all to spaces. I didn't see this locally since my editor shows tabs as 4 spaces, while Github shows them as 8.